seed/analysis_pipelines/better/client.py

```diff
@@ -197,7 +208,7 @@ def create_building(self, bsync_xml, better_portfolio_id=None):
         :param bsync_xml: str, path to BSync xml file for property
         :param better_portfolio_id: int | str, optional, if provided it will add the
             building to the portfolio
-        :returns: requests.Response building_id
+        :returns: tuple(int, list[str]), BETTER Building ID followed by list of errors
```

Comment:
before we would fail the entire pipeline if we failed to create a building, we should be more flexible and allow buildings to fail

Reply:
This makes sense to me.